modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -53,7 +53,8 @@
 													%>
 
 													<clay:link
-														buttonStyle="borderless"
+														borderless="<%= true %>"
+														button="<%= true %>"
```

Comment:
This API troubles me a little. Technically, it doubles `buttonStyle` into `button` and `displayStyle`.

In this particular case... this generates slightly different markup, going from `btn btn-borderless` to `btn btn-outline-borderless`. We could use `displayStyle="borderless"` here but I feel that might be wrong...

Reply:
If in doubt, try to stick as close to the React API as possible? We seem to have a pretty good correspondence with [`ClayButton` properties](https://clayui.com/docs/components/button.html):

- `alert` (bool)
- `block` (bool)
- `borderless` (bool)
- `displayType` (primary, secondary, link, unstyled)
- `monospaced` (bool)
- `outline` (bool)
- `small` (bool)

And [`ClayLink`](https://clayui.com/docs/components/link.html):

- `borderless` (bool)
- `button` (bool, or object with `block, `monospaced`, `small` bools).
- `monospaced`
- `outline`
- `displayType` (primary, secondary, unstyled)

Reply:
Yeah, that's what I did here... it's not very ergonomic to pass maps from jsp and we can't overload the tag setters (bool or map), so this is the next best thing I could come up with :(